Makefile

```diff
@@ -58,10 +58,16 @@ test-integration:
 
```

Comment:
tests/swarmtest/swarmtest.go was the only file with an integration build tag - think we can remove test-integration Make target. Otherwise lgtm